Gemfile

```diff
@@ -24,7 +24,7 @@ gem "curb", "~> 0.8"
 
 gem "nokogiri", "~> 1.6"
 
-gem "rails_12factor"
+gem "rails_12factor", group: :production
```

Comment:
well I prefer to run staging and production in the same env, and just use different env vars for db's etc